### PR TITLE
Issue/26 - order-in-model -> All not supported structured should be at the bottom

### PR DESCRIPTION
**Pull request by michalsnik**
#26

### CODE REVIEW

File: rules/order-in-models.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ function getOrderValue(property) {
     val = 30;
   } else if (ember.isMultiLineFn(property)) {
     val = 40;
+  } else {
+    val = 50;
   }
```
Comment (bardzusny) [inline rules/order-in-models.js]: Just took a look, seeing some discrepancy between styleguide description (https://github.com/netguru/ember-styleguide#organize-your-models) and actual functionality here. Perhaps worth looking into.

File: test/order-in-models.js [hunk 2]
```diff
@@ -49,6 +49,10 @@ eslintTester.run('order-in-models', rule, {
     {
       code: 'export default DS.Model.extend(TestMixin, TestMixin2, {mood: computed("health", "hunger", function() {\n})});',
       parserOptions: {ecmaVersion: 6, sourceType: "module"},
+    },
+    {
+      code: 'export default DS.Model.extend({ a: attr("string"), b: belongsTo("c", { async: false }), convertA(paramA) { \n } });',
+      parserOptions: {ecmaVersion: 6, sourceType: "module"},
     }
```
Comment (bardzusny) [inline test/order-in-models.js]: See above comment - more like a feature request, but for example seems like a good idea to differentiate between public/private methods, etc.

File: rules/order-in-models.js (hunk 1 above)
Comment (michalsnik) [inline rules/order-in-models.js]: Yeah, I updated docs.

File: test/order-in-models.js (hunk 2 above)
Comment (michalsnik) [inline test/order-in-models.js]: I'll rather evolve this rule in further PR's :)

**Pull request merged**